Wrappers/Python/ccpi/viewer/utils/conversion.py

```diff
@@ -649,120 +632,344 @@ def FillOutputPortInformation(self, port, info):
         return 1
 
     def GetStoredArrayShape(self):
-        return self.__StoredArrayShape
+        return self._StoredArrayShape
 
     def GetFileHeaderLength(self):
-        return self.__FileHeaderLength
+        return self._FileHeaderLength
 
     def GetBytesPerElement(self):
-        return self.__BytesPerElement
+        return self._BytesPerElement
 
     def GetBigEndian(self):
-        return self.__BigEndian
+        return self._BigEndian
 
     def GetIsFortran(self):
-        return self.__IsFortran
+        return self._IsFortran
 
     def GetOutputVTKType(self):
-        return self.__OutputVTKType
+        return self._OutputVTKType
 
     def GetNumpyTypeCode(self):
-        return self.__NumpyTypeCode
+        return self._NumpyTypeCode
 
     def SetStoredArrayShape(self, value):
         if not isinstance(value, tuple):
             raise ValueError('Expected tuple, got {}'.format(type(value)))
         if len(value) != 3:
```

Comment:
Here we are assuming 3D data, but we could also support 2D and 4D datasets in the future

Reply:
relevant issue here: https://github.com/vais-ral/CILViewer/issues/140